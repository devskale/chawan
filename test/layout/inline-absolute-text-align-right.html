<!DOCTYPE html>
<div style="width: 22ch; background: red">
test
<span style="position: relative">
test test test
<span style="position: absolute; top: 1em">
where am I?
