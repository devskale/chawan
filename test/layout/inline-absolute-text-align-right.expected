[48;2;255;0;0mtest test test test   [49m
                   where
                   am I?
